Bluepill-cli/Bluepill-cli/Simulator/SimulatorMonitor.m

```diff
@@ -248,7 +248,11 @@ - (void)stopTestsWithErrorMessage:(NSString *)message forTestName:(NSString *)te
     if (![[self.device stateString] isEqualToString:@"Shutdown"] && !self.config.testing_NoAppWillRun) {
         [BPUtils printInfo:ERROR withString:@"Will kill the process with appPID: %d", self.appPID];
         NSAssert(self.appPID > 0, @"Failed to find a valid PID");
-        [BPUtils printInfo:ERROR withString:[BPUtils runShell:[NSString stringWithFormat:@"/usr/bin/sample %d", self.appPID]]];
+        NSDateFormatter *dateFormatter=[[NSDateFormatter alloc] init];
+        [dateFormatter setDateFormat:@"yyyy-MM-dd_HH-mm-ss"];
+        NSString *sampleLogFile = [NSString stringWithFormat:@"/tmp/sampleLog_%@.txt", [dateFormatter stringFromDate:[NSDate date]]];
```

Comment:
We have `config.outputDir` for this. I would use that instead of `/tmp`

Reply:
Also isn't the PID of the app a better differentiator than the date? Or maybe both?

Reply:
+1 to both